src/modules/esl-utils/misc/uid.ts

```diff
@@ -1,10 +1,11 @@
+const SEQUENCE_KEY = Symbol('__esl_sequences__');
```

Comment:
```suggestion
const SEQUENCE_KEY = (window.Symbol || String)('__esl_sequences__');
```
As mentioned above, we can't declare symbols without fallback until 5th version with official IE support drop released